src/arrayMethodSort.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    a.toString() > b.toString()) {
```

Comment:
Should create function outside. It will be easier to read your code 
```suggestion
  [].__proto__.sort2 = function(compareFunction = defaultFunction) {
```